bgpd/rfapi/rfapi.c

```diff
@@ -137,7 +137,9 @@ int rfapi_get_response_lifetime_default(void *rfp_start_val)
 int rfapi_is_vnc_configured(void *rfp_start_val)
 {
 	struct bgp *bgp = rfapi_bgp_lookup_by_rfp(rfp_start_val);
-	return bgp_rfapi_is_vnc_configured(bgp);
+	if (bgp_rfapi_is_vnc_configured(bgp) == 0)
```

Comment:
so I'm missing the subtlety of this change.

bgp_rfapi_is_vnc_configured()  returns 0 if we are configured and ENXIO if not, here we check for the same things.  What am I missing?

Reply:
This is prep for adding more error codes in the next change...